packages/gen-ai/package.json

```diff
@@ -1,17 +1,20 @@
 {
   "private": true,
-  "name": "@odh-dashboard/llama-stack-modular-ui",
-  "description": "LLama Stack Modular UI plugin.",
+  "name": "@odh-dashboard/gen-ai",
+  "description": "Gen AI plugin.",
   "version": "0.0.0",
+  "exports": {
+    "./extensions": "./frontend/src/odh/extensions.ts"
+  },
```

Comment:
This is not needed. The extensions are contributed from this module via the module federation config.